homeassistant/components/agent_dvr/.translations/en.json

```
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "title": "Agent DVR",
```

```suggestion
  "title": "Agent DVR",
  "config": {
```

Core change #33850 moved the "title" key to the top level of the JSON earlier today.